test/runtime/runtime-kotlin/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Gradle-specific files
+build/
+
+# Generated Src files
+src/main/kotlin/com/mycompany/app/generic
```

Comment:
I would just use `src/main`, unless you explicitly know that you will have src files that is not generated 🙂

Reply:
done !!